### ISSUE TITLE
Lab 01 - Manage Microsoft Entra ID Identities- Not able to create new Entra Tenant under godeploy labs and not user as well

### ISSUE THREAD

**Issue by vikasmct** [MicrosoftLearning/AZ-104-MicrosoftAzureAdministrator#811]
# Module: 01
#### Step: 00

Description of issue - Not able to create New Entrar ID and not able to create new users.

Repro steps:

1.
![Untitled](https://github.com/MicrosoftLearning/AZ-104-MicrosoftAzureAdministrator/assets/134141333/418227c4-ebce-4a29-993d-2e2862269554)

1.
1.

**Comment by renaudwangler** [comment]
Hello.
I use the same labs (goDeploy) in my courses and had no problem to create the new tenant : did you launch the correct lab ?
You should open a goDeploy ticket since your problem is not related to this repository (you're supposed to use godeploy "lab guide" to use their lab environment : they make small changes to instructions from this repository).

Hope this helps ;)

**Comment by vikasmct** [comment]
> Hello. I use the same labs (goDeploy) in my courses and had no problem to create the new tenant : did you launch the correct lab ? You should open a goDeploy ticket since your problem is not related to this repository (you're supposed to use godeploy "lab guide" to use their lab environment : they make small changes to instructions from this repository).
> 
> Hope this helps ;)

Hi, Microsoft stopped this from last 2 days onwards and i raised this ticket to MS learning team , go deploy and MS product team.

this is serious issue and yes when i esclated it deep level , i have checked 10 times and all students also not able to do same thing.

**Comment by staleycyn** [comment]
The Create Tenant task is marked optional in the GitHub. As an alternative use the **[interactive lab simulation](https://mslabs.cloudguides.com/guides/AZ-104%20Exam%20Guide%20-%20Microsoft%20Azure%20Administrator%20Exercise%201)** to view the steps. <br>

**Comment by vikasmct** [comment]
Godeploy lab azure ad , we have no rights to do so where we can do labs ?

**Comment by labwhisperer** [comment]
The messaging on the create tenant blade is a bit confusing and could do with being linked to an explanation. Having done some research and testing, I can confirm that a paid-for license is now required to create new tenants.

I first attempted to purchase a P1 license in the M365 portal, but the system would only allow me to sign up for a trial. I signed up for the trial, assigned the license to my user but this failed to enable the "Microsoft Entra ID" option on the create tenant blade.

Back in the M365 portal, I removed the P1 trial, purchased a P2 license, assigned it to my user and refreshed the create tenant blade. This enabled the "Microsoft Entra ID" option, allowing me to create a new tenant.

It seems like this is a change Microsoft are gradually rolling out as go deploy seemed affected but Skillable, initially, was not.

In summary:

- It is/will be impossible to create new tenants for non-paying customers (users)
- Using a trial P1/P2 license will not work
- A P1/P2 must be _purchased_ and assigned in order to create new tenants
- I am not sure about other licenses

**Comment by vikasmct** [comment]
This is simple.

1. MS Stopped creating new tenant across all subscription .
2.  Default go deploy subscription we dont have any permission , even not to create user.

My Q to MS is how we do labs , either provide atleast 5 days option or remove this official labs

**Comment by ppedvStefanO** [comment]
Same Problem here. Students are unable to create new Tenant (ALH is GoDeploy) , GoDeploy Support told me i have to Contact Microsoft because they are unable to do anything against it. And with my MCT Subscription im also unable to create new Tenants with the same message.

**Comment by bwoodbridge** [comment]
It's not just creating a new tenant; even worse is not being able to create
any new users in the next part of the lab, and that's probably because the
default LabUser (not just one but I've tried several of them from sever
re-starts of this lab), does not have a subscription.

I've even tried using these steps of trying to create a new user in other
labs, and there also it fails but with a different error, "Validation
failed: invalid or missing information" on the step to Review and Create

A week ago Skillable support said they had escalated this to MS, but did
not have any resolution to report after 5 days.

We need help on this, even a workaround, please.  A new Az-104  begins
tomorrow.

**Comment by vikasmct** [comment]
Microsoft has no answer and I also escalated to Microsoft, Done AZ-104 with bad things.

That why I raised this to GitHub so that everyone are aware and no last-moment shock.

No support no result.

**Comment by edbakermct** [comment]
Hi

Was there ever any response to this?

It is really poor to still have this lab in the course if nbot. I have opened a new issue as there doesnt seem to ever have been an official response and the changelog doesnt mention it at all.

**Comment by renaudwangler** [comment]
> Hi
> 
> Was there ever any response to this?
> 
> It is really poor to still have this lab in the course if nbot. I have opened a new issue as there doesnt seem to ever have been an official response and the changelog doesnt mention it at all.
(have a look at the updated instructions : the first lab is officially replaced by the simulation, the lab steps are provided for any student willing to do it on another environement).
